src/app/(proper_react)/redesign/(authenticated)/user/welcome/Onboarding.test.tsx

```diff
@@ -210,3 +212,29 @@ it("passes the axe accessibility test suite on step 3", async () => {
   const { container } = render(<ComposedOnboarding stepId="findExposures" />);
   expect(await axe(container)).toHaveNoViolations();
 });
+
+it("shows a condensed version of the onboarding skipping step “Get started”", () => {
+  const ComposedOnboarding = composeStory(Onboarding, Meta);
+  render(<ComposedOnboarding skipInitialStep />);
+  const proceedButton = screen.getByRole("button", {
+    name: "Find exposures",
+  });
+  expect(proceedButton).toBeInTheDocument();
+});
+
+it("does not navigate back to step 1 of the onboarding", async () => {
```

Comment:
```suggestion
it("does not navigate back to step 1 of the onboarding when `skipInitialStep` is set", async () => {
```

Reply:
Updated in [6d87164](https://github.com/mozilla/blurts-server/pull/3541/commits/6d871645da55f045dded8c1b34133f708f94a102).